README.md

```diff
@@ -35,8 +34,8 @@ latter command to remove the container, add the `--rm` flag to the above `docker
 2. `yarn eth:node` - to start hardhat network
 3. `yarn eth:prepare-dao-contracts-for-hardhat` - to download the DAO contract sources locally. You need to run this
    only when running for the first time.
-4. (Optional) Modify the pool contract `EPOCH_LENGTH` variable from `1 weeks` to `1 minute` to speed up testing. You can
-   find this constant inside `dao-contracts/packages/pool/contracts/StateUtils.sol`
+4. (Optional) Modify the pool contract `EPOCH_LENGTH` variable from `1 weeks` to `1 minutes` to speed up testing. You
+   can find this constant inside `dao-contracts/api3-dao/packages/pool/contracts/StateUtils.sol`
```

Comment:
Minor: The path is still `dao-contracts/packages/pool/contracts/StateUtils.sol` on the `production` branch

Reply:
Fixed, thanks 🙏